### PR TITLE
Update buildkite

### PR DESCRIPTION
**Pull request by CarloLucibello**
replaces #88

### CODE REVIEW

**Comment by darsnack** [comment]
I think you need to increase the timeout.

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -1,10 +1,32 @@
...
+          dirs:
+            - src
+            - ext
     agents:
       queue: "juliagpu"
       cuda: "*"
+    env:
+      GROUP: "CUDA"
+    if: build.message !~ /\[skip tests\]/
     timeout_in_minutes: 120
```
Comment (CarloLucibello) [inline .buildkite/pipeline.yml:19]:
```suggestion
    timeout_in_minutes: 180
```

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
-          version: "1"
-      - JuliaCI/julia-test#v1: ~
+          version: "{{matrix.julia}}"
+      - JuliaCI/julia-test#v1:
+          test_args: "--quickfail"
+      - JuliaCI/julia-coverage#v1:
+          codecov: true
+          dirs:
+            - src
+            - ext
```
Comment (ToucheSir) [inline .buildkite/pipeline.yml:12]:
```suggestion
```
This is breaking coverage on CI

**Comment by CarloLucibello** [comment]
Julia 1 is passing, I'm happy enough. Merge?

**Review by darsnack — APPROVED** [review]
Julia 1.6 failure is weight loading issue that @theabhirath  mentioned.

**Pull request merged**